README.md

```diff
@@ -159,17 +217,14 @@ exec($(lab completion xonsh))
 source <(lab completion zsh)
 ```
 
-## What about [GLab](https://github.com/profclems/glab)?
+# Contributing
 
-Both [glab] and `lab` are open-source tools with the same goal of bringing GitLab to your command line and simplifying the developer workflow. In many ways `lab` is to [hub], what [glab] is to [gh].
+See the [contribution guide](CONTRIBUTING.md).
 
-`lab` aims to feel familiar to a `git` user and leverages `git` to power many of it's commands. `glab` will feel more familiar to `gh` users and in turn is more interactive and likely more beginner friendly for that reason.
+# What about [GLab](https://github.com/profclems/glab)?
```

Comment:
Should we continue to mention Glab?  I think this section can also be dropped.

Reply:
Yes, I thought about that too and was about to propose its removal. I don't really think this paragraph really helps one to understand what's the difference between the tool. And it refers to `hub` which is an already deprecated tool.
I'll remove it.

Reply:
@prarit , based on Zaq's comment below I let this section and even enhanced it to get closer to what is said about _lab_ in _glab_'s repo.